scripts/templates/api.h.mako

```diff
@@ -92,6 +92,9 @@ typedef enum ${th.make_enum_name(n, tags, obj)}
     %endfor
 
 } ${th.make_enum_name(n, tags, obj)};
+%if th.type_traits.is_flags(obj['name']):
+${th.make_flags_bitmask(n, tags, obj, meta)}
```

Comment:
I guess this is where the tempate documentation for the flags would go.

Reply:
👍